twine-core/src/transient/transient_simulation.rs

```diff
@@ -66,8 +66,16 @@ where
     /// successful initialization via [`TransientSimulation::new`].
     pub fn step(&mut self, dt: Time) -> Result<(), C::Error> {
         let last = self.history.last().unwrap();
-        let next = self.integrator.step(&self.component, last, dt)?;
-        self.history.push(next);
+        let next_input = self.integrator.integrate_state(&self.component, last, dt)?;
+
+        // Apply controls logic? See: <https://github.com/isentropic-dev/twine/issues/101>
```

Comment:
So I understand. This implies that every transient simulation will need a controller. Some transient simulations may not need a controller, so in those cases we can just provide a "unit" controller that simply returns the same input spit out by the integrator.